crates/sui-protocol-config/src/lib.rs

```diff
@@ -454,6 +524,24 @@ impl ProtocolConfig {
         // To change the values here you must create a new protocol version with the new values!
         match version.0 {
             1 => Self {
+                max_tx_in_batch: Some(10),
+                max_modules_in_publish: Some(8),
+                max_arguments: Some(16),
+                max_type_arguments: Some(128),
+                max_type_argument_depth: Some(16),
+                // TODO: does this need to be big? Will we have cause for big blobs
```

Comment:
@oxade @dariorussi just wanted to highlight the question I left for you in this comment